themes/theme-ios11/pages/Login/index.jsx

```diff
@@ -176,13 +173,19 @@ class Login extends Component {
 
   /**
    * Renders the component.
-   * @return {JSX}
+   * @return {JSX.Element}
    */
   render() {
     const isCheckout = this.props.redirect?.location === CHECKOUT_PATH;
+    const {
+      login, showErrors, loginError, password, passwordError,
+    } = this.state;
+    const { isLoading, isDisabled } = this.props;
+    const hasLoginError = loginError.length > 0;
```

Comment:
i've changed the logic a bit, since both properties initially have a value. i've combined it with the `showErrors` prop, like it's done at other places in the component.